### DISCUSSION TITLE
[Projects Beta] More granular access over the project (beta) structure (project name, attributes, views)

### DISCUSSION THREAD

**Discussion by KristinaKlimashevskaia** [community/community#7478] [Projects and Issues]
Currently write access/role allows to create/remove/update columns and options, views, field values, change project beta name and even close the project
Read access allows nothing to edit
Would be great (as a min requirement) to have a middle role where collaborators are allowed to add items to the project and to edit field values, with no control over project structure (like adding new fields, renaming existing) and no control over views and filters  (just read them)

As a further enhancement: would be nice to have more granular control:
1. in cut of the fields
2. for views, e.g. common views set by admins and available as read-only for collaborators vs private views that collaborators can create and manage regardless of access level, not visible to others, unless specifically shared with individuals or teams.

**Comment by astout** [comment]
[just a comment, not an answer]

Yes! I am exploring the Projects beta for sprint planning and would like to have numerous different views. Some views where only admins can change the structure as you say, and other views where certain members can perhaps modify the structure and change status of issues, etc. i.e. we could create a view for each developer in the project so they can see their active tasks in the current sprint. They can modify this view to their liking, then they can switch to the project overview that has all the issues, but aren't able to modify anything from this view.
